contrib/lib/src/compression.rs

```diff
@@ -0,0 +1,133 @@
+//! This module provides brotli ang gzip compression for all non-image
```

Comment:
`and`?

Reply:
Sorry, I don't understand your comment.

Reply:
I think that you made a typo here. Shouldn't it read `This module provides brotli and gzip compression for all non-image`?

Reply:
Right, thanks. These silly typos are the most difficult to see. I have changed it and it will be fixed in the next commit.